frontend/src/modules/lobby/components/InvitationLink.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <div class="invitation-link-section">
+    <b-container fluid>
+      <b-row align-v="center" class="justify-content-md-center">
+        <b-col md="auto">Invitation Link: </b-col>
+        <b-col md="auto"><input id="invitation-link" type='text' v-model="inviteLink" readonly></b-col>
+        <b-col md="auto"><b-button class="copy-link-button" variant="success" v-on:click="copyInviteLink">Copy Link</b-button></b-col>
+      </b-row>
+    </b-container>
+  </div>
+</template>
+
+<script>
+
+export default {
+  name: "InvitationLink",
+  data: function() {
+    return {
+        // inviteLink: window.location.href
+        inviteLink: "http://" + document.location.hostname + ":" + document.location.port + "/#/" + this.stage + "/" + this.lobbyID
```

Comment:
Have you looked into using this.$router to get he inviteLink?

Reply:
I did some google when i was doing this part and i can't really get this to work. I will try to do more digging. but yea i think what i have here is super janky.